interface/main/dated_reminders/dated_reminders_add.php

```diff
@@ -23,10 +23,11 @@
   $fake_register_globals=false;
   $sanitize_all_escapes=true;     
     
-    require_once("../../globals.php"); 
-    require_once("$srcdir/htmlspecialchars.inc.php");  
-    require_once("$srcdir/dated_reminder_functions.php"); 
+  require_once("../../globals.php"); 
+  require_once("$srcdir/htmlspecialchars.inc.php");  
+  require_once("$srcdir/dated_reminder_functions.php"); 
   require_once("$srcdir/formatting.inc.php");
+  require_once("$srcdir/headers.inc.php");
```

Comment:
Are there any calls that require this? Looks like datepicker is used in this code

Reply:
I saw datepicker in a number of things when reviewing today.  Dropping that for HTML5 is a different project, but the call_required_libraries is used immediately below for bootstrap.

Reply:
Ah, probably not. Will delete this.